.github/workflows/ofrak-type-ci.yml

```
@@ -0,0 +1,619 @@
+name: OFRAK Type, IO, Patch Maker, and Core CI
+
+on:
+  pull_request:
+    paths:
+      - 'ofrak_type/**'
+      - 'ofrak_io/**'
+      - 'ofrak_patch_maker/**'
+      - 'pytest_ofrak/**'
+      - 'ofrak_core/**'
+      - 'frontend/**'
+      - '.github/workflows/ofrak-type-ci.yml'
+  push:
+    branches:
+      - feature/windows_tests
+    #      - master
+#    paths:
+#      - 'ofrak_type/**'
+#      - 'ofrak_io/**'
+#      - '.github/workflows/ofrak-type-ci.yml'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    shell: bash  # NOTE: On Windows, this uses Git Bash which provides Unix compatibility (for example for ofrak-io's StreamCapture
```

```suggestion
    shell: bash -euxo pipefail {0}
```

See comments in other workflow.